src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -20,12 +22,14 @@
     private static final Logger logger = Logger.getLogger(StandardSlackService.class.getName());
 
     private String host = "slack.com";
+    private String baseUrl;
     private String teamDomain;
     private String token;
     private String[] roomIds;
 
-    public StandardSlackService(String teamDomain, String token, String roomId) {
+    public StandardSlackService(String baseUrl, String teamDomain, String token, String roomId) {
         super();
+        this.baseUrl = baseUrl;
```

Comment:
Trim off any potential spaces.  This should cover any case where `﻿StandardSlackService` is used.